index.bs

```diff
@@ -536,7 +536,7 @@ To support obtaining assertions via {{CredentialsContainer/get()|navigator.crede
 {{PublicKeyCredential}}'s [=interface object=]'s implementation of the
 <dfn for="PublicKeyCredential" method>\[[Create]](options)</dfn> method allows scripts to call
 {{CredentialsContainer/create()|navigator.credentials.create()}} to request the creation of a new [=credential key pair=]
-and {{PublicKeyCredential}}, managed by an [=authenticator=]. The user agent will prompt the user for [=user consent|consent=].
+and {{PublicKeyCredential}}, managed by an [=authenticator=].
```

Comment:
rather than simply deleting this, perhaps we should replace it with a forward-ref to the place in the spec where this is discussed?